packages/trie/src/trie/trie.ts

```diff
@@ -799,9 +799,9 @@ export class Trie {
   // (i.e. the Trie is not correctly pruned)
   // If this method returns `true`, the Trie is correctly pruned and all keys are reachable
   async verifyPrunedIntegrity(): Promise<boolean> {
-    const root = this.root().toString('hex')
+    const roots = [this.root().toString('hex'), this.hash(ROOT_DB_KEY).toString('hex')]
```

Comment:
```suggestion
    const roots = [this.root().toString('hex'), this.appliedKey(ROOT_DB_KEY).toString('hex')]
```

Should this not be the applied key? If trie is not hashed then this returns the wrong `ROOT_DB_KEY`.